scripts/docs/generate-transformations.ts

```diff
@@ -0,0 +1,301 @@
+
+{{< figure src="/static/img/docs/transformations/rename-by-regex-after-7-3.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+`}
+
+${`### ${transformationDocsContent['rowsToFields'].name}
+${transformationDocsContent['rowsToFields'].helperDocs}
+`}
+
+${`### ${transformationDocsContent['seriesToRows'].name}
+${transformationDocsContent['seriesToRows'].helperDocs}
+`}
+
+${`### ${transformationDocsContent['sortBy'].name}
+${transformationDocsContent['sortBy'].helperDocs}
+`}
+
+${`### ${transformationDocsContent['spatial'].name}
+${transformationDocsContent['spatial'].helperDocs}
+`}
+
+${`### ${transformationDocsContent['timeSeriesTable'].name}
+${transformationDocsContent['timeSeriesTable'].helperDocs}
+`}
+
+{{% docs/reference %}}
+[Table panel]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/table"
+[Table panel]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/table"
+
+[Data frames]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins/introduction-to-plugin-development/data-frames"
+[Data frames]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins/introduction-to-plugin-development/data-frames"
+
+[Calculation types]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/calculation-types"
+[Calculation types]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/calculation-types"
+
+[sparkline cell type]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/table#sparkline"
+[sparkline cell type]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/table#sparkline"
+
+[Heatmap panel]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/heatmap"
+[Heatmap panel]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/heatmap"
+
+[configuration file]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#configuration-file-location"
+[configuration file]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#configuration-file-location"
+
+[Time series panel]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series"
+[Time series panel]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series"
+
+[feature toggle]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#feature_toggles"
+[feature toggle]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#feature_toggles"
+{{% /docs/reference %}}
+`;
+
+// `process.stdout.write(template)` was not functioning as expected, but `console.log` also writes to the standard output.
+console.log(template);
```

Comment:
Interesting; could be. I'll look into it.

Reply:
Looked into it more this morning; it doesn't seem to be an issue with `tsc`, since `ts-node` also has the same issue: the `@grafana/tsconfig/base.json` file which is extended in the `tsconfig.json` can't be found by either transpiler, and therefore can't identify the typing for the node `process` object. Oddly enough, I can't find said `base.json` in the node modules either. I've tried cleaning and reinstalling grafana's deps to clear any caches and artifacts, but still no go. I think we'll just have to leave this for now.

Reply:
Ah interesting, oh well it's unfortunate but not a big deal.